packages/adapters/bitcoin/src/connectors/OKXConnector.ts

```diff
@@ -147,20 +200,16 @@ export class OKXConnector extends ProviderEventEmitter implements BitcoinConnect
   }
 
   public async switchNetwork(_caipNetworkId: CaipNetwork['caipNetworkId']): Promise<void> {
-    const connector = OKXConnector.getWallet({
-      requestedChains: this.requestedChains,
-      requestedCaipNetworkId: _caipNetworkId
-    })
-
-    if (!connector) {
-      throw new Error(`${this.name} wallet does not support network switching`)
-    }
-
-    this.unbindEvents()
-    this.wallet = connector.wallet
-
     try {
-      await this.connect()
+      const currentWallet = this.getWallet()
+      if (currentWallet) {
+        this.unbindEvents({ wallet: currentWallet })
+      }
+      const chain = this.chains.find(c => c.caipNetworkId === _caipNetworkId)
+      if (!chain) {
+        throw new Error('OKXConnector:switchNetwork - chain is undefined')
```

Comment:
There are incosistencies between the error messages, some of them has previx like `[connector][method] - xyz`, some of them not -  maybe we should add these to our `ErrorsUtil` and use that util to keep them aligned?

Reply:
Too many specific errors to add to utils imo. I'll change this one to match the others. 
I think this is something larger we need to standardzie codebase wide